.circleci/config.yml

```diff
@@ -223,7 +223,9 @@ jobs:
           name: Build calypso-strings.pot
           when: always
           command: |
-            npm run translate
+            CALYPSO_ENV=production NODE_ENV=build_pot npm run build-client
+            sudo apt install gettext
+            find build/.i18n-calypso -name \*.pot | msgcat -f- gutenberg-strings.pot -o calypso-strings.pot
```

Comment:
Would it make sense to define this step somewhere else in the toolchain, so it's e.g. easier to run it locally?

Arguably, we want to run this everytime the two relevant babel plugins have been used to extract strings into individual `.pot` files (which happens during the `build` step, if the environment is set to production). So it might make sense to define this as a `postbuild` (or `postbuild-client`?) script in `package.json` (with an environment check). We could then drop this line from `.circleci/config.yml`, since the translations would be automatically post-processed by the `build[-client]` step.

(I'll give that a try to verify.)

Reply:
```diff
diff --git a/package.json b/package.json
index ba07dba2ce..e4e5708adf 100644
--- a/package.json
+++ b/package.json
@@ -232,6 +232,7 @@
     "prebuild-server": "mkdirp build",
     "build-server": "cross-env-shell CALYPSO_SERVER=true NODE_PATH=$NODE_PATH:server:client:. webpack --display-error-details --config webpack.config.node.js",
     "build-client": "cross-env-shell CALYPSO_CLIENT=true NODE_PATH=$NODE_PATH:server:client:. node --max_old_space_size=8192 ./node_modules/webpack/bin/webpack.js --display-error-details",
+    "postbuild-client": "find build/.i18n-calypso -name *.pot | msgcat -f- gutenberg-strings.pot -o calypso-strings.pot",
     "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
     "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
     "sdk": "node bin/sdk-cli.js",
```

seems to work

Reply:
CircleCI config change:

```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index bc19c8df37..a0d91e1fbc 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -223,9 +223,8 @@ jobs:
           name: Build calypso-strings.pot
           when: always
           command: |
-            CALYPSO_ENV=production NODE_ENV=build_pot npm run build-client
             sudo apt install gettext
-            find build/.i18n-calypso -name \*.pot | msgcat -f- gutenberg-strings.pot -o calypso-strings.pot
+            CALYPSO_ENV=production NODE_ENV=build_pot npm run build-client
             mv calypso-strings.pot "$CIRCLE_ARTIFACTS/translate"
       - run:
           name: Build New Strings .pot
```

(untested)

Reply:
Thanks for the idea. I'll just actually change the `translate` command in the first place.